### ISSUE TITLE
FSE: ensure appropriate Template preview image alt text

### ISSUE THREAD

**Issue by getdave** [Automattic/wp-calypso#33697]
See `p4vt7e-4Q-p2#comment-180`.

We should ensure

* Alt text is pulled through for the Template preview images
* Appropriate alt text is written to describe each Template. This should be no more than 2 sentences.

Note: this cannot proceed until 

1. Template Pages are uploaded to the site
2. Preview images have been added to the correct location (see FG)
3. Static mapping of Templates -> Previews has been added

**Comment by getdave** [comment]
This is now complete
